openhtf/util/logs.py

```diff
@@ -55,22 +55,24 @@ def MyRandomMethod():
 import sys
 import traceback
 from datetime import datetime
-from openhtf import util
-import gflags
 
+from openhtf import util
+from openhtf.util import argv
 from openhtf.io import test_record
 
 
-FLAGS = gflags.FLAGS
-
-gflags.DEFINE_enum('verbosity',
-                   'warning', ['debug', 'info', 'warning', 'error', 'critical'],
-                   'Console log verbosity level (stderr).')
-gflags.DEFINE_boolean('quiet', False, "If True, don't output logs to stderr.")
-gflags.DEFINE_string('log-file', '', 'Filename to output logs to, if any.')
-gflags.DEFINE_enum('log-file-level', 'warning', ['debug', 'info', 'warning',
-                                                 'error', 'critical'],
-                   'Logging verbosity level for log file output.')
+LEVEL_CHOICES = ['debug', 'info', 'warning', 'error', 'critical']
+DEFAULT_LEVEL = 'warning'
+ARG_PARSER = argv.ModuleParser()
```

Comment:
See comment in `__init__.py`, I think we should use the `StoreInModule()` mechanism here too for consistency.